sources/platform/actors/development/programming_interface/basic_commands.md

```diff
@@ -332,7 +360,6 @@ await Actor.exit();
 
 ```python
 # 😔 Custom handlers are not supported in the Python SDK yet.
```

Comment:
We miss the Python here, is it correct? @vdusek / @janbuchar

Reply:
It is, you cannot subscribe to `exit` in Python SDK